Client/core/ServerBrowser/CServerList.cpp

```diff
@@ -371,29 +371,20 @@ std::string CServerListItem::Pulse(bool bCanSendQuery, bool bRemoveNonResponding
     }
     else
     {
-        // Poll the socket
-        sockaddr_in clntAddr;
-        int         addrLen = sizeof(clntAddr);
-        int         len = recvfrom(m_Socket, szBuffer, SERVER_LIST_QUERY_BUFFER, MSG_PARTIAL, (sockaddr*)&clntAddr, &addrLen);
-        int         error = WSAGetLastError();
-        if (len >= 0)
+        // Poll the socket and parse the packet

```

Comment:
Why are you changing the server list for Discord Rich Presence here? I'd prefer this to be in a separate pull request tbh.

Reply:
I can make it in another PR but this is necessary for Rich Presence, server won't share its name with clients and I use query to fetch the name, which was a mess and duplicating the code was painful and messy.